api/nodes/serializers.py

```diff
@@ -459,6 +473,18 @@ def get_registration_links_count(self, obj):
                 count += 1
         return count
 
+    def get_linked_by_nodes_count(self, obj):
+        return NodeRelation.objects.filter(child=obj, is_node_link=True).select_related('parent').exclude(parent__type='osf.collection').exclude(parent__type='osf.registration').count()
```

Comment:
Do you need to filter out deleted nodes linking to here, or is that handled elsewhere?

Reply:
This is the same logic used to calculate the count currently. My assumption here is that if a corresponding node is deleted, the NodeRelation object no longer exists. Will double check

Reply:
@brianjgeiger, after double-checking it looks like (in prod as well) the analytics page will show the count and the deleted projects that used to link to that node. I'm assuming that is not something we wanna keep on doing, but would be changing functionality

Reply:
I'm pretty sure, but am verifying with Product.